api/client/run.go

```diff
@@ -188,10 +188,15 @@ func (cli *DockerCli) CmdRun(args ...string) error {
 	}()
 
 	//start the container
-	if _, _, err = readBody(cli.call("POST", "/containers/"+createResponse.ID+"/start", nil, nil)); err != nil {
+	var obj []byte
+	if obj, _, err = readBody(cli.call("POST", "/containers/"+createResponse.ID+"/start", nil, nil)); err != nil {
```

Comment:
nit but can we avoid the var declaration and split in `obj, _, err := ...` and `if err != nil {}` otherwise `obj`?